### ISSUE TITLE
Excessive guilt

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#10185]
**Preferred term label:**
Excessive guilt

**Synonyms**
Shame?, Guilt

**Definition (free text, please give PubMed ID)**
Guilt that is excessive in that it is based on an incorrect judgment (that one is responsible for a negative outcome) or is out of proportion to the committed offense. Here we define guilt as the negative feeling that accompanies the belief that one is responsible for a negative outcome or event.

**Parent term (use hpo.jax.org/app)**
Abnormal emotion (HP:0100851)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Depression
https://www.omim.org/entry/608516

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by bcoleman96** [comment]
At first, I thought this term should broadly be under Abnormal thought, however, many of the definitions I see describe it as an emotion connected to a thought. A connected Cognitive distortion would be personalization (which is not yet a term), but it seems clear that personalization is a thought that leads to feelings of excessive guilt. On the other hand, one could argue that guilt is only excessive if it is in relation to an abnormal thought. A counter to this would be if one could experience feelings of guilt that are out of proportion to an offense without misinterpreting the event.

The DIGS question where this is relevant is: "Were you feeling guilty or that you were a bad person?" The second portion of this question clearly points to a belief which seems to be more essential to the diagnostic context. I might argue that the feeling of guilt is nearly indistinguishable from sadness except for the associated thoughts. If this is the case I would revise the above to focus on the self-blaming thoughts leading to excessive "sadness".

**Comment by pnrobinson** [comment]
@bcoleman96 
added as HP:6000011
